scene/gui/scroll_container.cpp

```diff
@@ -371,6 +371,9 @@ void ScrollContainer::update_scrollbars() {
 	Size2 hmin = h_scroll->get_combined_minimum_size();
 	Size2 vmin = v_scroll->get_combined_minimum_size();
 
+	if (!scroll_h) hmin.height = 0;
+	if (!scroll_v) vmin.width = 0;
```

Comment:
Maybe it would be cleaner to do this?
```cpp
Size2 hmin;
Size2 vmin;
if (scroll_h)
	hmin = h_scroll->get_combined_minimum_size();
if (scroll_v)
	vmin = v_scroll->get_combined_minimum_size();
```